lib/pages/chat/chat_pinned_events/pinned_events_style.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+class PinnedEventsStyle {
+  static const double maxHeight = 48;
+  static const double minHeightIndicator = 12;
+  static const double maxWidthIndicator = 2.0;
+  static const double maxLengthIndicator = 5.0;
+  static const double iconListSize = 24;
+
+  static double calcHeightIndicator(int length) {
+    if (length < maxLengthIndicator) {
```

Comment:
should handle the case length == 0

Reply:
<img width="554" alt="Screenshot 2023-11-07 at 16 05 31" src="https://github.com/linagora/twake-on-matrix/assets/99852347/59c78a87-e0ce-4efc-85ab-0d42dad26630">

If PinnedEvents is empty => will not be in this case